src/components/Input/Input.jsx

```diff
@@ -0,0 +1,13 @@
+import "./Input.scss";
+
+export default function Input({ name, type = "text", value, placeholder }) {

```

Comment:
```suggestion
export default function Input({ name, value, placeholder, type = "text" }) {
```


It is a personal decision, but I like to put at the end of the parameters these ones that have a default param.

Reply:
got it